<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MazeGUI</class>
 <widget class="QMainWindow" name="MazeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MazeGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="setBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rowlabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>row :</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rowspinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="collabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>column :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generatebtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>GENERATE</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="colspinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clearbtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="controlBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>191</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="HTSlabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>How to Solve?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DFSbtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>DFS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BFSbtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>BFS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextstepbtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prestepbtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>572</y>
      <width>901</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>901</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>rowspinBox</tabstop>
  <tabstop>colspinBox</tabstop>
  <tabstop>DFSbtn</tabstop>
  <tabstop>BFSbtn</tabstop>
  <tabstop>nextstepbtn</tabstop>
  <tabstop>prestepbtn</tabstop>
  <tabstop>generatebtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
